packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -0,0 +1,177 @@
+import IconMinimize from 'terra-icon/lib/icon/IconMinimize';
+import IconMaximize from 'terra-icon/lib/icon/IconMaximize';
+import IconChevronUp from 'terra-icon/lib/icon/IconChevronUp';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import Header from 'terra-clinical-header';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+import './ActionHeader.scss';
+
+const propTypes = {
+  /**
+   * Text to be displayed as the title in the header bar
+   **/
+  title: PropTypes.string,
+
+  /**
+   * Callback function for when the close button is clicked. The back button will not display if this is not set.
+   * On small viewports a back button will be displayed instead of a close button when a separate onBack callback is not set.
+   **/
+  onClose: PropTypes.func,
+
+  /**
+   * Callback function for when the back button is clicked. The back button will not display if this is not set.
+   **/
+  onBack: PropTypes.func,
+
+  /**
+   * Callback function for when the expand button is clicked.
+   * The expand button will not display if this is not set or on small viewports.
+   * Only the expand button will be rendered if onExpand and onMinimize are set.
+   **/
+  onExpand: PropTypes.func,
+
+  /**
+   * Callback function for when the minimize button is clicked.
+   * The minimize button will not display if this is not set or on small viewports.
+   * Only the expand button will be rendered if both onExpand and onMinimize are set.
+   **/
+  onMinimize: PropTypes.func,
+
+  /**
+   * Callback function for when the next button is clicked. The previous-next button group will not display if neither this or onPrevious are set.
+   **/
+  onNext: PropTypes.func,
+
+  /**
+   * Callback function for when the previous button is clicked. The previous-next button group will not display if neither this or onNext are set.
+   **/
+  onPrevious: PropTypes.func,
+
+  /**
+   * Child element to be displayed on the right end of the header.
+   **/
+  children: PropTypes.element,
+};
+
+const defaultProps = {
+  title: '',
+  onClose: null,
+  onBack: null,
+  onExpand: null,
+  onMinimize: null,
+  onNext: null,
+  onPrevious: null,
+  children: null,
+};
+
+const ActionHeader = ({
+  title,
+  onClose,
+  onBack,
+  onExpand,
+  onMinimize,
+  onPrevious,
+  onNext,
+  children,
+  ...customProps
+}) => {
+  const attributes = Object.assign({}, customProps);
+  const closeButton = onClose ? <Button icon={<IconClose />} onClick={onClose} /> : null;
+  const backButton = onBack ? <Button icon={<IconLeft />} onClick={onBack} /> : null;
+
+  let closeButtonSmall;
+  let backButtonSmall;
+  if (onClose && !onBack) {
+    backButtonSmall = <Button icon={<IconLeft />} onClick={onClose} />;
+    closeButtonSmall = null;
+  } else {
+    closeButtonSmall = closeButton;
+    backButtonSmall = backButton;
+  }
+
+
+  let expandButton;
+  if (!backButton) {
```

Comment:
The description says the expand button would not be displayed unless onExpand and onMinimize is set. Does this get checked somewhere?